tests/ut/backend/test_backend_handler.py

```
@@ -0,0 +1,102 @@
+        handler = _backend_core.get_handler()
+        assert handler.get_pto_target_arch() == "a2a3"
+        assert handler.requires_gm_pipe_buffer() is True
+
+    def test_global_handler_matches_configured_backend_950(self):
+        _backend_core.set_backend_type(_backend_core.BackendType.Ascend950)
+        handler = _backend_core.get_handler()
+        assert handler.get_pto_target_arch() == "a5"
+        assert handler.requires_vto_c_fractal_adapt() is True
+
+    def test_global_handler_raises_when_not_configured(self):
+        with pytest.raises(Exception):
```

Ruff (0.15.10)

[warning] Do not assert blind exception: `Exception`

(B017)